src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -245,6 +247,49 @@ def apply_quantization_status(model: Module, status: QuantizationStatus):
         model.apply(compress_quantized_weights)
 
 
+def expand_targets(
```

Comment:
because this is only used for sparsity cases where we don't have a scheme per module applied through , I would update the func name to reflect this.

Reply:
Done!